src/services/ImageRepository.h

```diff
@@ -20,37 +20,65 @@ enum class ImageType
 class ImageReference
 {
 public:
-    ImageReference() {}
-    ImageReference(ImageType nType, const std::string& sName)
+    ImageReference() noexcept = default;
+    explicit ImageReference(ImageType nType, const std::string& sName) noexcept
     {
         ChangeReference(nType, sName);
     }
 
-    ImageReference(const ImageReference& source)
-        : m_nType(source.m_nType), m_sName(source.m_sName)
+    ImageReference(const ImageReference& source) noexcept :
+        m_nType{ source.m_nType }, m_sName{ source.m_sName }
```

Comment:
Copy constructor/assignment isn't required to be `noexcept `by the standard but since the defaulted copy constructor was `noexcept`, this was `noexcept `as well.

This was all for `AchievementOverlay`. I ignored the other classes for later.